server/priv/marketing/blog/2025/07/08/server-fcl.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Tuist Server Source Code Is Now Public"
+category: "product"
+tags: ["product"] 
+excerpt: "We're thrilled to announce that the Tuist Server is now source available. After years of building open source tools for the mobile development community, we're taking a significant step toward greater transparency while ensuring sustainable development."
+author: pepicrft
+---
+
+At Tuist, we are huge advocates of openness. We started Tuist as an open source Xcode project generator CLI and eventually built a business around the awesome foundation we’ve been developing for more than 7 years. As part of that effort, we’ve gifted the community open source solutions like [Noora](https://github.com/tuist/noora), which was featured on [Swift’s new website](https://www.swift.org/), [XcodeProj](https://github.com/tuist/xcodeproj), which many CLI tools depend on, and [XcodeGraph](https://github.com/tuist/xcodegraph), which we extracted from our project generation CLI and made available to everyone.
+
+Unfortunately, we couldn’t open everything in the early days of the company because we hadn’t figured out where we could capture value. This left us with an itch. We knew this would be temporary and that we’d figure out a path back to openness, so I’m thrilled to announce that the Tuist Server is now [source available](https://github.com/tuist/tuist/tree/main/server).
+
+## Why we’re opening the source code
```

Comment:
I wonder if we should have one more section with "What changes for me" or something like that to:
- explicitly mention that only the server will be under the FCL, the rest of the codebase, including the CLI, will stay MIT.
- highlight the features of FCL - code is source available, we accept contributions (and we'll work on a guide to make contributions easier for the communty), but organizations still need to pay for hosting the server for commercial purposes